amxmodx/meta_api.cpp

```diff
@@ -1730,6 +1738,14 @@ C_DLLEXPORT	int	Meta_Detach(PLUG_LOADTIME now, PL_UNLOAD_REASON	reason)
 	g_plugins.clear();
 	g_langMngr.Clear();
 
+	ArrayHandles.clear();
+	TrieHandles.clear();
+	TrieIterHandles.clear();
+	TrieSnapshotHandles.clear();
+	DataPackHandles.clear();
+	TextParsersHandles.clear();
+	GameConfigHandle.clear();
```

Comment:
Next time these have to be extended, I'd move them into a big helper function to avoid repeating everything.

Reply:
I thought about same thing, yes but there's few strange things in code, which r makes duplication more preferably.